README.adoc

```diff
@@ -188,208 +188,80 @@ Dec 02 18:00:01.093 DEBG registered endpoint, path: /producers, method: POST, lo
 ...
 ----
 
-Once everything is up and running, you can use `curl` directly to hit either of the servers.  But it's easier to use the `oxapi_demo` wrapper (see below).
+Once everything is up and running, you can use `curl` directly to hit either of
+the servers.  But it's easier to use the [`oxide` CLI](https://docs.oxide.computer/cli/manual) (see below).
 
 == Docker image
 
 This repo includes a Dockerfile that builds an image containing the Nexus and sled agent.  There's a GitHub Actions workflow that builds and publishes the Docker image.  This is used by the https://github.com/oxidecomputer/console/[console] project for prototyping, demoing, and testing.  This is **not** the way Omicron will be deployed on production systems, but it's a useful vehicle for working with it.
 
 == Quick demo
 
-There's a small demo tool called `./tools/oxapi_demo` that provides a slightly friendlier interface than `curl`, with the same output format.  To use the demo, the `node` and `json` programs should be installed and available in the users PATH:
+The `oxide` CLI can be installed from the [latest
+release](https://github.com/oxidecomputer/cli/releases).
```

Comment:
Are we publishing releases for illumos?

Reply:
I couldnt get cargo cross to work for illumos but maybe @jclulow can, its easy to turn on if it compiles

Reply:
https://github.com/oxidecomputer/cli/blob/main/Makefile#L25

Reply:
I'm reading through https://github.com/cross-rs/cross - I don't think illumos is a supported target  by that project.

Manually, I checked out the CLI repo and tried building the `x86_64-unknown-illumos` target via cross compilation, and am hitting a variety of linker errors (which I think makes sense; it's trying to use my host (Linux) linker).

On the other hand, I can totally check out the CLI repo and build it manually on my Helios machine. If cross-compilation doesn't work, we could add a buildomat job to natively compile + publish the CLI tool?

Reply:
yep for sure i already have a matrix for the others on release https://github.com/oxidecomputer/cli/blob/main/.github/workflows/make-release.yml#L10

Reply:
I have opened a PR: oxidecomputer/cli#117

This should produce an `oxide` binary for illumos systems.  Is there a way, from GitHub Actions, to poll on the completion of a check run from outside GitHub Actions?

Reply:
dope! hmm not sure, but since I'm uploading things to GCS maybe the release stuff could be changed to wait for everything to be in the bucket before generating things, idk

Reply:
illumos releases https://github.com/oxidecomputer/cli/releases/tag/v0.1.0